README.md

```diff
@@ -14,7 +14,7 @@ of kiwix.
 Most of the compilation steps are handled by the kiwix-build.py
 script.
 
-This script has been tested of Fedora 23 and Ubuntu 16.10
+This script has been tested on Fedora 23, Ubuntu 16.10 and Debian 8
```

Comment:
I understand.
Maybe, we could add another sentence to say that Debian 8 is not officially supported, but has been reported to work after adding ~/.local/bin in the PATH ?